.circleci/config.yml

```diff
@@ -78,7 +78,7 @@ test_and_cover: &test_and_cover
       - run: composer update -n --prefer-dist
 
       - run: |
-          [ -f /usr/local/lib/php/extensions/no-debug-non-zts-20200930/xdebug.so] || pecl install xdebug
+          [ -f /usr/local/lib/php/extensions/no-debug-non-zts-20190902/xdebug.so] || pecl install xdebug
```

Comment:
It looks like I typo'ed this in a few projects. However, what i'd expect is that this would only be a suboptimal use of the cache (since this will always run), not a segfault.

Reply:
That makes sense. I was totally guessing with this. Bummer that it's so intermittent at the moment. Probably just need to ignore it for a bit.